.github/workflows/pack.yml

```diff
@@ -12,6 +12,21 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v2
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            pppy/osu-web
+          # generate Docker tags based on the following events/attributes
+          tags: |
+            type=raw,value=latest,enable=${{ !contains(github.ref_name, '-') }}
+            type=raw,value=${{ github.ref_name }}
+            type=sha
```

Comment:
do we need this when there's already the one with tag name?

Reply:
not really, but also why not? seems like a good practice

Reply:
there's already another one whenever this is run that's unique (the tag/version) so this will just be image of same thing. Alternatively just append the hash to the version name

Reply:
it's not pushing twice, just a tag as another way to reference the image (depending on the user's use case, don't have one atm).

not willing to append hash, trying to find the image for a specific version will be a pain otherwise.

Reply:
> it's not pushing twice, just a tag as another way to reference the image (depending on the user's use case, don't have one atm).

but it still fills up the listing with redundant images in no meaningful way since version and hash corresponds one to one - that's what git tag is after all (and we're not overriding tags afaik).

> not willing to append hash, referencing the image for a specific version will be a pain otherwise.

can use more specifics on this. At least it doesn't make any difference for looking up on docker hub.

Reply:
docker image tag is automatically referenced by the Helm chart's "appVersion" field, would rather not add the git SHA to it.

Reply:
I'm not sure how that matter but fine, I guess. Also I just noticed the versioning (`GIT_SHA`) used for current deployment is `version-buildid?-hash` (something like `20220427-3411-74c50df`)

Reply:
The sentry release need some clean ups (namely sending actual commit hash) so it's fine for now.